specification/src/main/asciidoc/platform/TransactionManagement.adoc

```diff
@@ -240,34 +239,34 @@ access a database, invoke an enterprise bean that accesses the same
 database as part of the same transaction, and, finally, commit the
 transaction.
 
-A Java EE product must provide a transaction
+A Jakarta EE product must provide a transaction
 manager that is capable of coordinating two-phase commit operations
 across multiple XA-capable JDBC databases. If a JDBC driver supports the
 Java Transaction API’s XA interfaces (in the _javax.transaction.xa_
```

Comment:
This should be `Jakarta Transactions`

Reply:
The javax.transaction.xa package remains with Java SE and is defined by the JTA XA spec,
not the Jakarta Transactions spec.